packages/web-components/fast-components/src/color-2/recipes/neutral-fill-input.ts

```diff
@@ -0,0 +1,24 @@
+import { Palette } from "../palette";
+import { Swatch } from "../swatch";
+import { directionByMode } from "../utilities/direction-by-mode";
+
+export function neutralFillInput(
+    palette: Palette,
+    reference: Swatch,
+    restDelta: number,
+    hoverDelta: number,
+    activeDelta: number,
+    focusDelta: number,
```

Comment:
Let's not carry the `focus` and `selected` colors forward in this update.

Selected was planned to be deprecated long ago because a component in a 'selected' mode should also respond to `hover` and `active` states, and therefore needs its own recipe. That's where `neutralFillToggle` came in.

And for `focus` all of our current offsets are exactly the same as `rest`, we haven't applied them in styles, and it shares the same issue with 'selected' components. A focused component can also be hovered or pressed.